### ISSUE TITLE
[ARM/Linux] Regression in 'JIT/Directed/throwbox/rethrow/rethrow.exe'

### ISSUE THREAD

**Issue by parjong** [dotnet/coreclr#5594]
Unittest 'JIT/Directed/throwbox/rethrow/rethrow.exe' failed with the following output:
```
before 1st try...catch
inside 1st try
inside 1st catch
terminate called after throwing an instance of 'PAL_SEHException'
```

**Comment by parjong** [comment]
This failure dosen't appear for the Release binary.

**Comment by parjong** [comment]
PR #5595 fixes this issue.

**Comment by parjong** [comment]
Closed via PR #5595
